src/main/java/org/jitsi/videobridge/Content.java

```diff
@@ -552,6 +546,25 @@ private String generateChannelID()
                     System.currentTimeMillis() + Videobridge.RANDOM.nextLong());
     }
 
+    /**
+     * @return a new channel ID, unique in the list of this {@link Content}'s
+     * channels.
+     */
+    private String generateUniqueChannelID()
```

Comment:
the old code was calling this as well, but i only see it added here, not removed from anywhere else?

Reply:
The old code calls generateChannelID (without "Unique")